ymer/src/main/java/com/avanza/ymer/support/JavaTimeLocalDateTimeReadConverter.java

```diff
@@ -15,13 +15,18 @@
  */
 package com.avanza.ymer.support;
 
-import org.springframework.core.convert.converter.Converter;
-import org.springframework.data.convert.ReadingConverter;
-
 import java.time.Instant;
 import java.time.LocalDateTime;
 import java.time.ZoneId;
 
+import org.springframework.core.convert.converter.Converter;
+import org.springframework.data.convert.ReadingConverter;
+
+/**
+ * Converter that converts from a Long to a {@link LocalDateTime}. This is a good choice if
```

Comment:
Perhaps Long should have @link as well? To be consistent

Reply:
Sure! Why not? Have added links to all the converter-classes in a separate commit.